data/Remnant Cognizance.txt

```diff
@@ -0,0 +1,626 @@
+			`As you finish unloading you spot Gavriil approaching. He gestures in greeting as he approaches. "Thanks for the delivery. We've got standard logistics routes set up now for the rest."  He pauses and fishes a datacube out of a pouch. "Could you take this to Prefect Taely on Viminal? She is overseeing some retrofits and will need this data.`
+				accept
+	on complete
+		conversation
+			`On viminal you easily locate the maintenance yards and eventually find Taely burried in a Pelican. She spots you approaching and frees herself from the machinery. "Greetings <first>. How is our station faring?" You reply that it is improving rapidly, and hand her the datacube. "Ah, good. The detailed chemical analysis and profiles." She slots it into her console and taps a few commands. "OK, I just need to incorporate these profiles into the system and make a few more adjustments. Meet me in the cafeteria in a couple hours."`
+
+
+
+mission "Remnant: Cognizance 11"
+	name "Travel with the <escort target ship> to pick up a shipment of atmosphere from Nasqueron and deliver it to <destination>."
+	description "Go meet up with escorts"
+	cargo equipment 8
+	source "Viminal"
+	stopover "Nasqueron"
+	destination "Esquiline"
+	npc accompany
+		government "Remnant"
+		personality timid launching escort
+		fleet
+			names "remnant"
+			cargo 8 equipment
+			variant
+				Pelican
+				Gascraft 4
+	to offer
+		has "Remnant: Cognizance 10: done"
+	on offer
+		conversation
+			`In the cafeteria you are met by the Taely and several other captains. "Thank you for joining us," she signs. "The goal of this mission is to travel to Nasqueron together with a cargo of equipment designed to harvest some atmosphere from the void sprite's worlds, and bring it back to Esquiline. The equipment on the gascrafts will collect, compress, and store the gas, and them you can offload it to specialy prepared tanks on the pelican. Then proceed to Esquiline and offload what you have collected to the station.`
+				accept
+	on stopover
+		dialog
+			`You cruise through the atmosphere at a relaxed pace, waiting while the equipment does its work. You see fewer Void Spritest than usual, many have scars. Once the equipment indicates that it has reached capacity you climb up to where the pelican is waiting in the high atmosphere, hook up the connections to offload the gas, then head back down into the depths of the planet. After several hours the captain of the pelican informs you that the tanks are full, and it is time to head to Esquiline.`
+	on complete
+		event "remnant: postverta reveal"
+		conversation
+			`You arrive at the hidden station on Esquiline, where the Gavriil is waiting to offload the atmosphere. His team quickly hooks up the gascraft up to ports on the bay walls and the equipment starts disgorging its contents into the system. After a few minutes they disconnect the ships, and you and the gascrafts head up to the pelican to refill your reservoirs and ferry it down to the station.`
```

Comment:
`You arrive at the hidden station on Esquiline, where Gavriil is waiting to offload the atmospherice gasses. His team quickly hooks up the gascraft up to ports on the bay walls and the equipment starts ejecting its contents into the system. After a few minutes, they disconnect the ships, and you and the gascrafts head up to the pelican to refill your reservoirs and ferry it down to the station.`

Reply:
Fixed.